addons/l10n_es_edi_facturae/data/uom_data.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <!-- UOM.UOM -->
+        <!-- Units -->
+        <record id="uom.product_uom_unit" model="uom.uom">
+            <field name="l10n_es_edi_facturae_uom_code">01</field>
```

Comment:
These codes do not correspond to a certain standard or is it only Facturae specific?